appuio/redis/templates/redis-node-statefulset.yaml

```diff
@@ -183,11 +179,7 @@ spec:
               command:
                 - sh
                 - -c
-                {{- if .Values.sentinel.enabled }}
-                - /health/ping_readiness_local.sh {{ .Values.slave.livenessProbe.timeoutSeconds }}
-                {{- else }}
-                - /health/ping_readiness_local_and_master.sh {{ .Values.slave.livenessProbe.timeoutSeconds }}
-                {{- end }}
```

Comment:
Again, why are we removing the distinction between setups which use sentinel and ones which don't?

Reply:
The template is [only ever interpreted in `sentinel` mode](https://github.com/appuio/charts/blob/e5c3053542a669df30d628aec17c23bcc4afe03b/appuio/redis/templates/redis-node-statefulset.yaml#L1). The if is superfluous.